hw/top_earlgrey/doc/_index.md

```diff
@@ -449,6 +449,21 @@ For more details see the [flash controller module specification]({{< relref "hw/
 Interconnecting the processor and peripheral and memory units is a bus network built upon the TileLink-Uncached-Light protocol.
 See the [OpenTitan bus specification]({{< relref "hw/ip/tlul/doc" >}}) for more details.
 
+#### Topology
+`top_earlgrey` has a two-level hierarchy for its bus network.
+Close to the CPU is the high-speed cluster, with low access latency.
+Farther from the CPU, through a low-speed bridge, is the low-speed cluster, with higher access latency.
+
+#### Performance
+
+The following table describes the typical number of CPU cycles from a transaction's launch to its completion for a device in the specified cluster:
```

Comment:
Maybe worth adding a caveat about the bus not being busy?

Reply:
Done. :)